assets/js/modules/thank-with-google/components/settings/SettingsView.js

```diff
@@ -29,18 +29,27 @@ import Data from 'googlesitekit-data';
 import DisplaySetting from '../../../../components/DisplaySetting';
 import { MODULES_THANK_WITH_GOOGLE } from '../../datastore/constants';
 import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import { getColorThemes } from '../../util/settings';
 const { useSelect } = Data;
 
 export default function SettingsView() {
 	const publicationID = useSelect( ( select ) =>
 		select( MODULES_THANK_WITH_GOOGLE ).getPublicationID()
 	);
+	const colorTheme = useSelect( ( select ) =>
+		select( MODULES_THANK_WITH_GOOGLE ).getColorTheme()
+	);
 
 	// Bail if the values aren't ready.
 	if ( publicationID === undefined ) {
 		return null;
 	}
 
+	const { name: colorName } =
+		getColorThemes()?.find(
```

Comment:
I don't see the optional chaining being necessary here, was this left in by accident?

```suggestion
		getColorThemes().find(
```

Reply:
It was added "just in case". Should work without it 👍